package.json

```diff
@@ -41,6 +41,10 @@
     "ts-node": "^10.1.0",
     "yargs": "^17.0.1"
   },
+  "hopr": {
+    "description": "master",
+    "environment_id": "master"
```

Comment:
Can this be overwritten at build-time or do we always need to change it in file?

Reply:
It's meant for manul update when releasing.

Reply:
ok, I guess its a good start